segmentstore/server/src/test/java/io/pravega/segmentstore/server/store/StreamSegmentServiceNoOpWriteOnlyTests.java

```diff
@@ -79,6 +87,13 @@ public void testEndToEnd() throws Exception {
     @Override
     @Test
     public void testEndToEndWithFencing() throws Exception {
-        endToEndProcessWithFencing(false);
+        endToEndProcessWithFencing(false, false);
+    }
+
+    @Override
+    @Test
+    @Ignore
```

Comment:
And here

Reply:
NoOpStorage does not support ChunkedSegmentStorage yet.